src/hotspot/share/logging/logDecorators.cpp

```diff
@@ -45,6 +49,13 @@ const char* LogDecorators::_name[][2] = {
 #undef DECORATOR
 };
 
+const LogDecorators::DefaultUndecoratedSelection LogDecorators::default_decorators[] = {
+#define UNDECORATED_DEFAULT(level, ...) LogDecorators::DefaultUndecoratedSelection(level, __VA_ARGS__),
+  UNDECORATED_DEFAULTS
+#undef UNDECORATED_TAGSET
+};
+const size_t LogDecorators::number_of_default_decorators = sizeof(default_decorators) / sizeof(LogDecorators::DefaultUndecoratedSelection);
```

Comment:
I think this reads better and is less error prone.
```suggestion
const size_t LogDecorators::number_of_default_decorators = ARRAY_SIZE(default_decorators);
```